code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -782,4 +782,55 @@
 	accuracy_mult = config.base_hit_accuracy_mult - config.med_hit_accuracy_mult
 	scatter = config.med_scatter_value
 	damage_mult = config.base_hit_damage_mult
-	recoil = config.med_recoil_value
\ No newline at end of file
+	recoil = config.med_recoil_value
+
+//-------------------------------------------------------
+
+//-------------------------------------------------------
+//SCOUT SHOTGUN
+
+/obj/item/weapon/gun/shotgun/merc/scout
+	name = "\improper ZX-76 assault shotgun"
+	desc = "The MIC ZX-76 Assault Shotgun, a dobule barreled semi-automatic combat shotgun with a twin shot mode. Has a 9 round internal magazine."
+	icon_state = "zx-76"
+	item_state = "zx-76"
+	origin_tech = "combat=5;materials=4"
+	fire_sound = 'sound/weapons/gun_shotgun_automatic.ogg'
+	current_mag = /obj/item/ammo_magazine/internal/shotgun/scout
+	attachable_allowed = list(
+						/obj/item/attachable/bayonet,
+						/obj/item/attachable/reddot,
+						/obj/item/attachable/verticalgrip,
+						/obj/item/attachable/angledgrip,
+						/obj/item/attachable/gyro,
+						/obj/item/attachable/flashlight,
+						/obj/item/attachable/extended_barrel,
+						/obj/item/attachable/compensator,
+						/obj/item/attachable/magnetic_harness,
+						/obj/item/attachable/lasersight,
+						/obj/item/attachable/attached_gun/flamer,
+						/obj/item/attachable/attached_gun/shotgun,
+						/obj/item/attachable/attached_gun/grenade,)
+	attachable_offset = list("muzzle_x" = 32, "muzzle_y" = 17,"rail_x" = 8, "rail_y" = 18, "under_x" = 24, "under_y" = 12, "stock_x" = 13, "stock_y" = 15)
+
+/obj/item/weapon/gun/shotgun/merc/scout/New()
+	. = ..()
+	var/obj/item/attachable/stock/scout/G = new(src)
+	G.flags_attach_features &= ~ATTACH_REMOVABLE
+	G.Attach(src)
+	update_attachable(G.slot)
+	G.icon_state = initial(G.icon_state)
+	if(current_mag && current_mag.current_rounds > 0)
+		load_into_chamber()
+
+/obj/item/weapon/gun/shotgun/merc/scout/set_gun_config_values()
+	fire_delay = config.scoutshottie_fire_delay
+	burst_amount = config.low_burst_value
+	burst_delay = 0.01 //basically instantaneous two shots
```

Comment:
```suggestion
	burst_delay = min_fire_delay
```
That value is `1`, which is 0.2 seconds in this code. Basically instant.

Reply:
Not fast enough.

Reply:
Have you tried it?

Reply:
Yes. 0.01 is fine.

Reply:
add a `no_fire_delay` var on combat_defines then, which other weapons could use.

Reply:
This is fine.

Reply:
Magic numbers in the contribution guidelines, and all.

Reply:
Yes, it's not fast enough.

Reply:
Then do as Ghommie suggested, I guess.

Reply:
I just did it for fuck's sake.